R/pipeline-helpers.R

```diff
@@ -167,3 +167,63 @@ unprotect_integer_cols <- function(d) {
   names(d) <- sub('^X([0-9]+)$', '\\1', names(d))
   d
 }
+
+
+#' gdp_deflator
+#'
+#' Calculate a gross domestic product (GDP) implicit price deflator between two years.
+#'
+#' @param year Year for which to calculate deflator
+#' @param base_year Base year in deflator calculation
+#' @details The GDP deflator is a measure of price inflation/deflation with respect to a specific base year.
+#' @note This returns slightly different values if \code{OLD_DATA_SYSTEM_BEHAVIOR} is TRUE.
+#' @return Deflator (GDP in \code{year} divided by GDP in \code{base_year}).
```

Comment:
#' @return GDP_deflator_ratio (GDP_deflator in \code{year} divided by GDP_deflator in \code{base_year}).